plugins/bulk_upload/filter/batch/BulkUploadUserEntryEngineFilter.php

```diff
@@ -0,0 +1,81 @@
+<?php
+/**
+ * This engine supports deletion of user entries based on the input filter
+ *
+ * @package plugins.bulkUploadFilter
+ * @subpackage batch
+ */
+class BulkUploadUserEntryEngineFilter extends BulkUploadEngineFilter
+{
+	const OBJECT_TYPE_TITLE = 'user entry';
+
+	/* get a list of objects according to the input filter
+	 *
+	 * @see BulkUploadEngineFilter::listObjects()
+	 */
+	protected function listObjects(KalturaFilter $filter, KalturaFilterPager $pager = null)
+	{
+		if($filter instanceof KalturaUserEntryFilter)
+		{
+			return KBatchBase::$kClient->userEntry->listAction($filter, $pager);
+		}
+
+		else
+		{
+			throw new KalturaBatchException("Unsupported filter: {get_class($filter)}", KalturaBatchJobAppErrors::BULK_VALIDATION_FAILED);
+		}
+	}
+
+	protected function createObjectFromResultAndJobData (KalturaBulkUploadResult $bulkUploadResult)
+	{
+

```

Comment:
IS this function entry because the task itself does delete and there is no object to return?
If so add a comment explaining this and return null as a response.